autupdate.sh

```diff
@@ -44,26 +44,36 @@ for item in $REPOS; do
   if [[ -f '.autoupdate/update' ]]; then
     .autoupdate/update
   else
+    if test -f '.circleci/config.yml' && grep -q ruby-rails .circleci/config.yml; then
+      latest=$(circleci orb info sul-dlss/ruby-rails | grep 'Latest:' | cut -d@ -f2)
+
+      sed -i -e "s/sul-dlss\/ruby-rails@.*/sul-dlss\/ruby-rails@$latest/" .circleci/config.yml &&
+        git add .circleci/config.yml &&
+        git commit -m "Update CircleCI orb"
+    fi
```

Comment:
This should filter out @sul-dlss/access-team repos, since they're not using CircleCI or our custom ruby-rails orb at this point.